### PR TITLE
feat[web-14]: add modal

### PR DESCRIPTION
**Pull request by GarnaA**
Added video modal

### CODE REVIEW

File: src/components/Modals/VideoModal/components/CloseButton/closeButton.module.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    background-color: var(--common-color-bg-inverse-primary);
+    border: none;
+    border-radius: 50%;
+    transition: transform 0.6s ease;
+    transform: rotate(45deg);
+}
+
+.button:hover{
+    transform: rotate(135deg);
+}
```
Comment (sofiiavoitkova) [inline src/components/Modals/VideoModal/components/CloseButton/closeButton.module.scss]: Add a newline at the end of the file.

File: src/components/Modals/VideoModal/components/CloseButton/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  )
+}
+
+Button.propTypes = {
+  onClose: T.func,
+}
+
+export default Button
+
+
```
Comment (sofiiavoitkova) [inline src/components/Modals/VideoModal/components/CloseButton/index.jsx]: Remove unnecessary empty lines for cleaner code.

File: src/components/Modals/VideoModal/index.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    window.addEventListener('keydown', close)
+
+    return () => window.removeEventListener('keydown', close)
+  }, [])
+
+  if (elementModal === null) return null
+
+
```
Comment (sofiiavoitkova) [inline src/components/Modals/VideoModal/index.jsx]: Remove unnecessary empty line.

File: src/components/Modals/VideoModal/index.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import { useEffect } from 'react'
+import T from 'prop-types'
+import { createPortal } from 'react-dom'
+import Button from './components/CloseButton'
+import styles from './modal.module.scss'
+
+const VIDEO_LINK = "https://www.youtube.com/embed/4Oj_fenobDw?enablejsapi=1"
+
+function Modal( isOpen, onClose, videoLink = VIDEO_LINK ) {
```
Comment (sofiiavoitkova) [inline src/components/Modals/VideoModal/index.jsx]: Destructure props in Modal to ensure clarity and proper usage.

**Review by AndrZakhara — APPROVED** [review]
fix and resolve existing conversations

**Review by MyHKo — APPROVED** [review]

**Review by sofiiavoitkova — APPROVED** [review]

**Pull request merged**